app/services/stripe_authorization_service.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module StripeAuthorizationService
+  FORBIDDEN_MERCHANT_CATEGORIES =
+    Set.new(
+      [
+        "betting_casino_gambling",
+        "cigar_stores_and_stands",
```

Comment:
Are we sure we want to block this? Cigar stores and tobacconists often sell electronics such as chargers or soft drinks, which we should allow.

Reply:
Fair point! Got this list from Mel. Let me double-check

Reply:
Did some internal digging and looks like we've seen legitimate transactions for `cigar_stores_and_stands` so will remove

Reply:
https://github.com/hackclub/hcb/pull/11405/commits/10133016ddd7037962c25c53ca9746aade82b7c9